cg/store/crud/read.py

```diff
@@ -456,6 +444,49 @@ def get_sample_lane_sequencing_metrics_by_flow_cell_name(
             flow_cell_name=flow_cell_name,
         ).all()
 
+    def get_illumina_metrics_entry_by_device_sample_and_lane(
+        self, run_id: str, sample_internal_id: str, lane: int
```

Comment:
Did we not agree to use something else than run_id? 
```suggestion
        self, device_internal_id: str, sample_internal_id: str, lane: int
```

Reply:
I completely missed that, you are right

Reply:
fixed